src/NUnitFramework/framework/Does.cs

```diff
@@ -63,20 +63,7 @@ public static FileOrDirectoryExistsConstraint Exist
         /// Returns a new <see cref="EqualConstraint"/> checking for the
         /// presence of a particular object in the collection.
         /// </summary>
-        public static EqualConstraint Contain(object expected)
-        {
-            return new ConstraintExpression().Some.EqualTo(expected);
-        }
-
-        /// <summary>
-        /// Returns a new ContainsConstraint. This constraint
-        /// will, in turn, make use of the appropriate second-level
-        /// constraint, depending on the type of the actual argument. 
-        /// This overload is only used if the item sought is a string,
-        /// since any other type implies that we are looking for a 
-        /// collection member.
-        /// </summary>
-        public static ContainsConstraint Contain(string expected)
+        public static ContainsConstraint Contain(object expected)
```

Comment:
I think we should keep the string overload for the sake of the XML docs, don't you?